spec/index.html

```diff
@@ -1729,9 +1739,9 @@ <h2>Entailment rules (Informative)</h2>
       to the set in all possible ways, to exhaustion.</li>
   </ol>
 
-  <p>Then we have the completeness result:</p>
+  <p>If these rules are complete, they would give rise to the following completeness result:</p>
 
-  <p class="fact">If S is RDF (RDFS) consistent, then S RDF entails (RDFS entails) E just
+  <p class="postulate">If S is RDF (RDFS) consistent, then S RDF entails (RDFS entails) E just
     when the <a>generalized RDF (RDFS) closure</a> of S towards E <a>simply entails</a> E. </p>
```

Comment:
Is there a strong reason not to state this as below?
```suggestion
  <p class="postulate">If S is RDF consistent, then S RDF entails E just
    when the <a>generalized RDF closure</a> of S towards E <a>simply entails</a> E. </p>
  <p class="postulate">Likewise, if S is RDFS consistent, then S RDFS entails E just
    when the <a>generalized RDFS closure</a> of S towards E <a>simply entails</a> E. </p>
```
I'm concerned about the parentheticals leading to misunderstanding.

Reply:
If there *is* strong argument for keeping the parenthetical, structure, I think the following would be more internally consistent.
```suggestion
  <p class="postulate">If S is RDF consistent (RDFS consistent), then S RDF entails (RDFS entails) E just
    when the <a>generalized RDF (RDFS) closure</a> of S towards E <a>simply entails</a> E. </p>
```

Reply:
Separately, I think `only` is better than `just` here --
```suggestion
  <p class="postulate">If S is RDF (RDFS) consistent, then S RDF entails (RDFS entails) E only
    when the <a>generalized RDF (RDFS) closure</a> of S towards E <a>simply entails</a> E. </p>
```